src/setup_mtc_place_from_pose.cpp

```diff
@@ -0,0 +1,227 @@
+#include <example_behaviors/setup_mtc_place_from_pose.hpp>
+
+#include <behaviortree_cpp/bt_factory.h>
+#include <moveit/task_constructor/stages.h>
+#include <moveit/task_constructor/solvers.h>
+#include <moveit/task_constructor/task.h>
+#include <moveit_msgs/msg/collision_object.hpp>
+#include <moveit_studio_behavior_interface/behavior_context.hpp>
+#include <moveit_studio_behavior_interface/check_for_error.hpp>
+#include <moveit_studio_behavior_interface/yaml_parsing_tools.hpp>
+#include <sensor_msgs/msg/joint_state.hpp>
+#include <tf2_eigen/tf2_eigen.hpp>
+#include <yaml-cpp/yaml.h>
+
+namespace
+{
+const auto kLogger = rclcpp::get_logger("SetupMTCPickFromPose");
+using MoveItErrorCodes = moveit_msgs::msg::MoveItErrorCodes;
+
+// Port names for input and output ports.
+constexpr auto kPortIDTask = "task";
+constexpr auto kPortIDGraspPose = "place_pose";
+
+// Behavior constants
+constexpr auto kWorldFrame = "/world";
+constexpr auto kArmGroupName = "manipulator";
+constexpr auto kEndEffectorGroupName = "gripper";
+constexpr auto kEndEffectorName = "moveit_ee";
+constexpr auto kHandFrameName = "grasp_link";
+constexpr auto kHandOpenName = "open";
+constexpr auto kHandCloseName = "close";
+constexpr auto kApproachDistance = 0.1;
+constexpr auto kPropertyNameTrajectoryExecutionInfo = "trajectory_execution_info";
+constexpr double kIKTimeoutSeconds = 1.0;
+constexpr int kMaxIKSolutions = 20;
+constexpr auto kSceneObjectNameOctomap = "<octomap>";
+constexpr auto kSceneObjectName = "object";
```

Comment:
```suggestion
```
We got rid of this in pick, so probably can get rid of it here (if we remove the detach stage)